mimic/rest/loadbalancer_api.py

```diff
@@ -54,10 +55,128 @@ def resource_for_region(self, region, uri_prefix, session_store):
         Get an :obj:`twisted.web.iweb.IResource` for the given URI prefix;
         implement :obj:`IAPIMock`.
         """
+        self.session_store = session_store
         lb_region = LoadBalancerRegion(self, uri_prefix, session_store,
                                        region)
         return lb_region.app.resource()
 
+    def _get_session(self, session_store, tenant_id):
+        """
+        Retrieve or create a new LoadBalancer session from a given tenant identifier
+        and :obj:`SessionStore`.
+
+        For use with ``data_for_api``.
```

Comment:
Oh, ugh, I see the code you copied it from.  Probably code I wrote.  This is terrible but it's something I should clean up later, so please feel free to disregard this comment for now.